db/seeds.rb

```diff
@@ -5,3 +5,8 @@
 #
 #   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
 #   Mayor.create(name: 'Emanuel', city: cities.first)
+
+User.create(email: 'admin@example.com',
+            password: 'password',
+            password_confirmation: 'password',
+            admin: true)
```

Comment:
Could you convert this to a `find_or_create_by(:email)`?  You can pass in a block to set the rest of the attributes.  There is a uniqueness constraint on the users email which will cause this to fail if it's run multiple times.  `find_or_create_by` avoids this issue if people need to load new seeds.